compiler/src/dotty/tools/dotc/transform/init/Semantic.scala

```diff
@@ -1531,22 +1525,22 @@ object Semantic {
    *  This is intended to avoid type soundness issues in Dotty.
    */
   def checkTermUsage(tpt: Tree, thisV: Ref, klass: ClassSymbol): Contextual[Unit] =
-    val traverser = new TypeTraverser {
-      def traverse(tp: Type): Unit = tp match {
+    val traverser = new TypeTraverser:
+      def traverse(tp: Type): Unit =
+        tp match
```

Comment:
What's the rule for indenting the cases of a match? Do they need to be indented more?